pipes/config.yaml

```diff
@@ -192,6 +192,27 @@ vphaser_max_bins: 10
 # allele freq > 0.005. If false, no filtering is performed.
 vcf_merge_naive_filter: false
 
+# Preprocessing method for reads for de novo assembly
+assembly_preproc_method: 'subsamp'
+
+# Method of de novo assembly
+assembly_de_novo_method: 'trinity'
+
+# Method of choosing the reference to use for scaffolding
```

Comment:
Right.  On this branch there's only one method choice  for each stage, since I just added placeholders.  Plan was to merge this branch and then make pull requests for additions of individual methods; these pull  requests would  then be less  likely to create merge conflicts  since they'd affect different placeholders.